rasa/core/tracker_store.py

```diff
@@ -129,6 +129,8 @@ def create(
             ConnectionError,
         ) as error:
             raise ConnectionException("Cannot connect to tracker store.") from error
+        except pymongo.errors.OperationFailure as error:
```

Comment:
why not just include it in the list above?

Reply:
I had a discussion with Tobias in the issue [thread](https://github.com/RasaHQ/rasa/issues/8726#issuecomment-847659020) which concluded that it would be confusing to label it as `ConnectionException`.

I also tried putting the code in the `MongoTrackerStore` constructor in a try-except block, but the unit test i added still failed, because the exception that got raised was still `OperationFailure` instead of `RasaException`. Not sure what I was doing wrong so I ended up putting exception handling in the factory method.

Reply:
I agree with Tobias that really these framework specific exceptions should be caught in their dedicated implementation classes. However I guess since we are already doing a few here it should be fine.